include/nil/blueprint/non_native_marshalling.hpp

```diff
@@ -165,7 +165,13 @@ namespace nil {
             else {
                 if constexpr (non_native_policy::ratio == 1) {
                     column_type<BlueprintFieldType> res;
-                    res.push_back(typename BlueprintFieldType::value_type(typename BlueprintFieldType::integral_type(input.data)));
+
+                    typename BlueprintFieldType::integral_type chopped_val =
+                        typename BlueprintFieldType::integral_type(
+                            typename NonNativeFieldType::integral_type(input.data)
+                        );
+
+                    res.push_back(typename BlueprintFieldType::value_type(chopped_val));
```

Comment:
Why not use mask in case of modular bits no native type < modular bits field type, like https://github.com/NilFoundation/zkllvm-assigner/blob/7fc7a6d6ce99e8df88099a6c3647d23b098f9380/include/nil/blueprint/non_native_marshalling.hpp#L169

Reply:
@CblPOK-git